src/main/java/com/box/sdk/BoxWebLink.java

```diff
@@ -277,7 +277,7 @@ protected void parseJSONMember(JsonObject.Member member) {
                     try {
                         this.linkURL = new URL(value.asString());
                     } catch (MalformedURLException e) {
-                        throw new BoxAPIException("Couldn't parse url for weblink", e);
+                        this.linkURL = null;
```

Comment:
Should we only do this if the value is an empty string? i.e. set linkURL to null if the url is missing, but still throw the exception if a url is present but malformed